lib/whitehall_importer/create_image_revision.rb

```diff
@@ -56,7 +56,11 @@ def create_blob_revision(temp_image)
     def normalise_image(file)
       upload_checker = Requirements::ImageUploadChecker.new(file)
       abort_on_issue(upload_checker.issues)
-      normaliser = ImageNormaliser.new(file)
+
+      raw_image = MiniMagick::Image.open(file.path)
+      raw_image.combine_options(&:strip)
```

Comment:
Does `raw_image.strip` have the same effect? https://www.rubyguides.com/2018/12/minimagick-gem

Reply:
Ah ha, you're absolutely correct! I've moved it all onto one line as it boiled down to fairly simple code 💥